UserProfile.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<SprintRecords> DevideBySprints(List<UserRecords> list)
+        {
+            DateTime date = list[0].Date;
+            int quarterNumber = (date.Month - 1) / 3 + 1;
+            DateTime firstDayOfQuarter = new DateTime(date.Year, (quarterNumber - 1) * 3 + 1, 1);
+            DateTime lastDayOfQuarter = firstDayOfQuarter.AddMonths(3).AddDays(-1);
+
+            var listtemp = new List<UserRecords>();
+            var sprint = new List<SprintRecords>();
+            foreach (var record in list)
+            {
+                if (firstDayOfQuarter < record.Date && record.Date < lastDayOfQuarter)
```

Comment:
What happens in the case here the record date is exactly on the first day of the quarter or the last? Unless I am mistaken, we would incorrectly group those records.